CodeGeneration/Sources/generate-swiftsyntax/templates/swiftparser/ParserTokenSpecSetFile.swift

```diff
@@ -93,15 +92,15 @@ let parserTokenSpecSetFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
                     SwitchCaseSyntax(
                       "case .\(raw: keyword.escapedName): return .keyword(.\(raw: keyword.escapedName))"
                     )
-                  case .token(let tokenText):
-                    let token = SYNTAX_TOKEN_MAP[tokenText]!
-                    if token.text != nil {
+                  case .token(let tokenKind):
+                    let caseName = tokenKind.tokenSpec.varOrCaseName
+                    if tokenKind.tokenSpec.text != nil {
                       SwitchCaseSyntax(
-                        "case .\(raw: token.varOrCaseName): return .\(raw: token.varOrCaseName)Token()"
+                        "case .\(raw: caseName): return .\(raw: caseName)Token()"
```

Comment:
Since you’re touching this, I think you can remove the `raw:` here. Looks like I forgot it when I added `varOrCaseName`.

Same 4 lines below